src/FileIO/README.md

```diff
@@ -0,0 +1,33 @@
+# File I/O
+
+The `FileIO` module provides basic file I/O operations.
+Right now, these are limited to reading bytes from a file, and writing bytes to a file.
+The API is intentionally limited in scope and will be expanded later.
+
+Unlike other modules in the `libraries` repo,
+the `FileIO` module will not compile or run correctly without a language-specific implementation file.
+Language-specific implementation files are provided for C#/.NET, Java, and Javascript.
+(Python and Golang support are planned.)
```

Comment:
Looks like Python is in fact supported? :)

Was Go skipped for lack of time, or a deeper reason?

Reply:
Python is *almost* supported, but the Python compilation of the `FileIO` module doesn't correctly import `DafnyLibraries`. When I manually edit the `FileIO_Compile.py` to `import DafnyLibraries` at the top level, then everything works as expected. For that reason I've included the `FileIO.py` implementation here even though it can't quite be tested automatically.

Reply:
Since we don't want to check in untested code, I've removed the Python bits.